project-5/main.py

```diff
@@ -111,24 +126,22 @@ def insert(self) -> None:
                 raise InsertError("cannot insert into table") from e
 
     def validation(self):
```

Comment:
lets change the function name to `validate_duplicate_columns`

Reply:
Done i used table validation because the function adds constraints to the table